activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-model/src/main/java/org/activiti/cloud/services/query/model/TaskVariableEntity.java

```diff
@@ -53,6 +55,8 @@ public class TaskVariableEntity extends AbstractVariableEntity {
 
     private String taskId;
 
+    private boolean ephemeral;
```

Comment:
In order to avoid the same problem we had with process variables the field should not use the primitive type but `Boolean` that allows null
```suggestion
    private Boolean ephemeral;
```
However, I think it's better to keep using the primitive type for the getter return `false` if it's `null` to avoid propagating nulls:
```java
public boolean isEphemeral() {
    return Boolean.TRUE.equals(this.ephemeral);
}
```
Recap:
- field uses `Boolean` (object)
- setter uses `Boolean` (object)
- getter returns `boolean` (primitive) to avoid propagating nulls and considers `null` as `false`